neqo-transport/src/tparams.rs

```diff
@@ -27,28 +27,72 @@ use crate::{
     Error, Res,
 };
 
-pub type TransportParameterId = u64;
-pub const ORIGINAL_DESTINATION_CONNECTION_ID: TransportParameterId = 0x00;
-pub const IDLE_TIMEOUT: TransportParameterId = 0x01;
-pub const STATELESS_RESET_TOKEN: TransportParameterId = 0x02;
-pub const MAX_UDP_PAYLOAD_SIZE: TransportParameterId = 0x03;
-pub const INITIAL_MAX_DATA: TransportParameterId = 0x04;
-pub const INITIAL_MAX_STREAM_DATA_BIDI_LOCAL: TransportParameterId = 0x05;
-pub const INITIAL_MAX_STREAM_DATA_BIDI_REMOTE: TransportParameterId = 0x06;
-pub const INITIAL_MAX_STREAM_DATA_UNI: TransportParameterId = 0x07;
-pub const INITIAL_MAX_STREAMS_BIDI: TransportParameterId = 0x08;
-pub const INITIAL_MAX_STREAMS_UNI: TransportParameterId = 0x09;
-pub const ACK_DELAY_EXPONENT: TransportParameterId = 0x0a;
-pub const MAX_ACK_DELAY: TransportParameterId = 0x0b;
-pub const DISABLE_MIGRATION: TransportParameterId = 0x0c;
-pub const PREFERRED_ADDRESS: TransportParameterId = 0x0d;
-pub const ACTIVE_CONNECTION_ID_LIMIT: TransportParameterId = 0x0e;
-pub const INITIAL_SOURCE_CONNECTION_ID: TransportParameterId = 0x0f;
-pub const RETRY_SOURCE_CONNECTION_ID: TransportParameterId = 0x10;
-pub const VERSION_INFORMATION: TransportParameterId = 0x11;
-pub const GREASE_QUIC_BIT: TransportParameterId = 0x2ab2;
-pub const MIN_ACK_DELAY: TransportParameterId = 0xff02_de1a;
-pub const MAX_DATAGRAM_FRAME_SIZE: TransportParameterId = 0x0020;
+#[allow(clippy::enum_clike_unportable_variant)]
```

Comment:
```suggestion
#[repr(u64)]
```

Suppressing the warning seems fine, but you could instead acknowledge the true size of this type.